codegen/smithy-ruby-codegen/src/main/java/software/amazon/smithy/ruby/codegen/generators/ErrorsGeneratorBase.java

```diff
@@ -74,7 +89,7 @@ public void render(FileManifest fileManifest) {
                 .call(() -> renderErrorCodeBody())
                 .closeBlock("end")
                 .call(() -> renderBaseErrors())
-                .call(() -> renderServiceModelErrors())
+                .call(() -> renderServiceModelErrors(new ErrorsVisitor()))
```

Comment:
Is there a reason we create and pass the visitor here rather than letting that method handle it?